src/app.module.ts

```diff
@@ -27,6 +27,29 @@ import { HealthCheckModule } from './health-check/health-check.module';
 import { DIDContactModule } from './modules/did-contact/did.contact.module';
 import { StatusListModule } from './modules/status-list/status-list.module';
 import { IPFSModule } from './modules/ipfs/ipfs.module';
+import { ValidationOptions } from 'joi';
+import { envVarsValidationSchema } from './env-vars-validation-schema';
+
+const validationOptions: ValidationOptions = {
+  stripUnknown: true,
+  allowUnknown: false,
+  abortEarly: false,
+};
+
+let config: DynamicModule;
+
+try {
+  config = ConfigModule.forRoot({
+    isGlobal: true,
+    // below is a workaround for E2E not executing because of https://github.com/energywebfoundation/ssi-hub/blob/c7191fd73778665fb9b26a36b0b0e1a5af8a2e33/e2e/app.e2e.spec.ts#L39
+    validationOptions: !process.env.E2E ? validationOptions : null,
```

Comment:
skipping validation in tests reduces introduces potential vulnerabilities
also we would better be more specific in which enironment validation is enabled
```suggestion
    validationOptions: process.env.PROD ? validationOptions : null,
```

Reply:
@JGiter 
I agree. However I would not call this vulnerabilities because this is not a validation of user input, but env variables. This is a workaround for the problem in E2E tests triggered by validation enabled. If we find another way to fix it, we can get rid of this workaround.